components/utilities/data/README.md

```diff
@@ -0,0 +1,83 @@
+following commands:
+
+1. `yarn get:auctions`
+2. `yarn update:auctions`
+
+Based on past behavior, it will sometimes be required to modify the cache manually. The team has
+noticed that Kusama will occasionally introduce a breaking api change or modifications to the
+dispatch queue before the auctions begins. The `getAuctions.js` script will not attempt to monitor
+changes to an auction while in the dispatch queue if it has already been added to the cache. This
+means if a `startBlock` was incorrectly added, it will not update with the appropriate auction
+information as the block is likely not the start of an auction. If this issue is detected, the
+automation routine will bail (displaying an error for the action) and not commit any changes until
+the discrepancy is resolved manually. New auctions can be added any time and changes can be manually
+appended. The following cache properties must be provided:
+
+1. `index` - auction number (count starts at 0)
+2. `startBlock` - the block an auction is expected to become active
+
+**The remaining properties in the cache are auto-generate and should not be modified as running
+`yarn update:auctions` will overwrite those changes.** This means when manually adding a new auction
+these are the only 2 properties that need to be specified. The remaining properties should still be
+including, providing `null` dates or blocks and
+`0x0000000000000000000000000000000000000000000000000000000000000000` for hash values. For example,
+if I wanted to manually add a new auction `(#41)` to the cache I would append the following object:
+
+```json
+  {
+    "index": 40,
+    "startBlock": 14411200,
+    "startHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "endPeriodBlock": null,
+    "endPeriodHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "biddingEndsBlock": null,
+    "biddingEndsHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "onboardStartBlock": null,
+    "onboardStartHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "onboardEndBlock": null,
+    "onboardEndHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "startDate": null,
+    "endPeriodDate": null,
+    "biddingEndsDate": null,
+    "onboardStartDate": null,
+    "onboardEndDate": null
+  },
+```
+
+Once this information is added and saved to the local json file, I can run `yarn update:auctions`
+which will attempt to see if any of the other properties can be updated. The final steps are to
+commit and push the changes to the main branch on the repo. This will ensure the
+[update-auction-schedules](https://github.com/w3f/polkadot-wiki/blob/master/.github/workflows/update-auction-schedules.yml)
+GitHub action will detect the new auction and continue to check for future updates daily (or at the
+interval specified in the automation `.yml` file).
```

Comment:
```suggestion
Once this information is added and saved to the local JSON file, I can run `yarn update:auctions`
which will attempt to see if any of the other properties can be updated. The final steps are to
commit and push the changes to the main branch on the repo. This will ensure the
[update-auction-schedules](https://github.com/w3f/polkadot-wiki/blob/master/.github/workflows/update-auction-schedules.yml)
GitHub action will detect the new auction and continue to check for future updates daily (or at the
interval specified in the automation `.yml` file).
```